modules/oxide/src/less/theme/components/toolbar-button/toolbar-split-button.less

```diff
@@ -15,25 +17,29 @@
   }
 
   .tox-split-button:hover {
-    box-shadow: 0 0 0 1px @toolbar-button-hover-background-color inset;
+    box-shadow: 0 0 0 2px @toolbar-split-button-focus-background-color inset;
+  }
+
+  .tox-split-button__chevron svg {
+    fill: @toolbar-button-chevron-color;
   }
 
   .tox-split-button:focus {
-    background: @toolbar-button-focus-background-color;
+    background: @toolbar-split-button-focus-background-color;
     box-shadow: @toolbar-button-focus-box-shadow;
-    color: @toolbar-button-focus-text-color;
+    color: @toolbar-split-button-focus-text-color;
+
+    svg {
+      fill: @toolbar-split-button-focus-text-color;
```

Comment:
This might be why your changes break the "current colour" line in the forecolor and backcolor toolbar buttons. Those use SVG `fill` styles to change dynamically:
https://github.com/tinymce/tinymce/blob/a81d5d3b7e3d9efd1126b202a1858ae2a9022640/modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts#L265-L269

Maybe your styles are impacting more elements than you expected? Try using the DOM inspector to see what's overriding the runtime fill styles.

Reply:
This also happens on 6-stable when using 'oxide-dark' skin

Reply:
I just figured this out. The underline bar is for showing the currently selected color (by calling setIconFill). So it works as expected